src/java.desktop/share/classes/javax/swing/plaf/nimbus/skin.laf

```diff
@@ -21646,6 +21649,10 @@
                <uiProperty name="tabOverlap" type="INT" value="-1"/>
                <uiProperty name="extendTabsToBase" type="BOOLEAN" value="true"/>
                <uiProperty name="useBasicArrows" type="BOOLEAN" value="true"/>
+               <uiProperty name="contentOpaque" type="BOOLEAN" value="true"/>
+               <uiProperty name="contentAreaColor" type="COLOR">
```

Comment:
SInce you are introducing those 3 property in NimbusLookANdFeel, is this defines needed in skin.laf?

Reply:
I am not sure about this. Should I add them in skin.laf ?

Reply:
> you have already added...i am asking it it's needed?

ohh yes, i messed up with other file.

> SInce you are introducing those 3 property in NimbusLookANdFeel, is this defines needed in skin.laf?

In NimbusLookAndFeel there is no such property defined. To keep JTabbedPane behavior consistent in Nimbus LAF, properties are added to skin.laf. Else the ContentArea color will be same as `JPanel color when non-opaque and JTabbedPane background color when opaque.`